### PR TITLE
autostart on insert & renderAt

### PR DESCRIPTION
**Pull request by bso-odoo**

### CODE REVIEW

File: addons/web/static/tests/public/interaction.test.js [hunk 1]
```diff
@@ -1746,6 +1746,49 @@ describe("insert", () => {
...
+            };
+            setup() {
+                this.renderAt(
+                    "web.TestSubInteraction1",
+                    {
+                        first: "one",
+                        second: "two",
+                    },
+                    this.el
+                ); // "beforeend"
```
Comment (romo-odoo) [inline addons/web/static/tests/public/interaction.test.js]: I don't think it is necessary to give the default value here

File: addons/web/static/tests/public/interaction.test.js [hunk 2]
```diff
@@ -1746,6 +1746,49 @@ describe("insert", () => {
...
+            static selector = "[data-which]";
+            dynamicContent = {
+                _root: {
+                    "t-att-x": () => "x",
+                },
+            };
+        }
+
+        const { core, el } = await startInteraction([Test, Test2], TemplateTest);
+        expect(core.interactions.length).toBe(3); // 1*Test + 2*Test2
```
Comment (romo-odoo) [inline addons/web/static/tests/public/interaction.test.js]: `expect(core.interactions).toHaveLength(3);`

File: addons/web/static/tests/public/interaction.test.js [hunk 3]
```diff
@@ -1746,6 +1746,49 @@ describe("insert", () => {
...
+        const testEl = el.querySelector(".test");
+        const subEls = testEl.querySelectorAll("[data-which][x=x]");
+        await click(subEls[1]);
+        await click(subEls[0]);
+        expect.verifySteps(["two", "one"]);
+        core.stopInteractions();
+        expect(testEl.querySelectorAll("[data-which]").length).toBe(0);
```
Comment (romo-odoo) [inline addons/web/static/tests/public/interaction.test.js]: `expect(core.interactions).toHaveLength(0);`

File: addons/web/static/tests/public/interaction.test.xml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
```
Comment (romo-odoo) [inline addons/web/static/tests/public/interaction.test.xml]: Is this blank line necessary ?

File: addons/web/static/tests/public/interaction.test.xml [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+    <t t-name="web.TestSubInteraction1">
+        <div class="sub1" t-att-data-which="first">Sub 1</div>
+        <div class="sub2" t-att-data-which="second">Sub 2</div>
+    </t>
+
```
Comment (romo-odoo) [inline addons/web/static/tests/public/interaction.test.xml]: Is this blank line necessary ?

File: addons/web/static/tests/public/interaction.test.js [hunk 6]
```diff
@@ -1746,6 +1746,49 @@ describe("insert", () => {
...
+                _root: {
+                    "t-att-x": () => "x",
+                },
+            };
+        }
+
+        const { core, el } = await startInteraction([Test, Test2], TemplateTest);
+        expect(core.interactions.length).toBe(3); // 1*Test + 2*Test2
+        const testEl = el.querySelector(".test");
+        const subEls = testEl.querySelectorAll("[data-which][x=x]");
```
Comment (romo-odoo) [inline addons/web/static/tests/public/interaction.test.js:1781]: `const subEls = el.querySelectorAll(".test [data-which][x=x]");`

Comment (bso-odoo) [reply addons/web/static/tests/public/interaction.test.js:1781]: I'd rather keep it in the variable than specify `.test` in both calls.

**Pull request merged**